packages/@coorpacademy-components/src/atom/link/index.js

```diff
@@ -77,7 +83,7 @@ class Link extends React.Component {
     const _hoverStyle =
       skinHover && hovered
         ? {
-            color: primarySkinColor
+            color: hoverSkinColor
```

Comment:
```suggestion
            color: hoverColor
```
It could just be this simple